test/clj/rems/test_browser.clj

```diff
@@ -24,14 +24,84 @@
+                                            :license/type :text
+                                            :organization {:organization/id "nbn"}
+                                            :license/title {:en "General Terms of Use"
+                                                            :fi "Yleiset käyttöehdot"
+                                                            :sv "Allmänna villkor"}
+                                            :license/text {:en (apply str (repeat 10 "License text in English. "))
+                                                           :fi (apply str (repeat 10 "Suomenkielinen lisenssiteksti. "))
+                                                           :sv (apply str (repeat 10 "Licens på svenska. "))}})
+        wfid (test-helpers/create-workflow! {:type :workflow/default :title "Default workflow" :handlers ["handler" "developer"]})
+        decider-wf (test-helpers/create-workflow! {:actor "owner"
+                                                   :organization {:organization/id "nbn"}
+                                                   :title "Decider workflow"
+                                                   :type :workflow/decider
+                                                   :handlers ["carl" "handler"]})
+        form (test-data/create-all-field-types-example-form! "owner" {:organization/id "nbn"} "Form")
+        simple-form (test-helpers/create-form! {:actor "owner"
+                                                :organization {:organization/id "nbn"}
+                                                :form/title "Simple form"
+                                                :form/fields [{:field/title {:en "Simple text field"
+                                                                             :fi "Yksinkertainen tekstikenttä"
+                                                                             :sv "Textfält"}
+                                                               :field/optional false
+                                                               :field/type :text
+                                                               :field/max-length 100
+                                                               :field/privacy :private}]})
+        res-id1 (test-helpers/create-resource! nil)
+        item-id1 (test-helpers/create-catalogue-item! {:form-id form :workflow-id wfid :title {:en "Default workflow" :fi "Oletustyövuo"
+                                                                                               :sv "sv"} :resource-id res-id1})
+        app-id (test-helpers/create-draft! "applicant" [item-id1] "draft")]
+    (test-helpers/create-workflow-licence! wfid link)
+    (test-helpers/create-workflow-licence! wfid text)
+    (test-helpers/create-workflow-licence! decider-wf link)
+    (test-helpers/create-workflow-licence! decider-wf text)
+    (test-helpers/submit-application app-id "applicant"))
```

Comment:
whew, that's quite a bit of test-data. now that we see it all explicitly we can later pare it down or move it to the individual tests

Reply:
True, and removing test-data-fixture from standalone-fixture was smalled than removing it from api-fixture is going to be...